app/experimenter/experiments/models/nimbus.py

```diff
@@ -155,11 +152,14 @@ class Filters:
         )
         IS_PAUSE_QUEUED = Q(
             status=NimbusConstants.Status.LIVE,
-            publish_status=NimbusConstants.PublishStatus.IDLE,
+            status_next=NimbusConstants.Status.LIVE,
+            publish_status=NimbusConstants.PublishStatus.APPROVED,
+            # TODO: EXP-1325 Need to check for something else here for end enrollment?
```

Comment:
This comment was Les' and I'm not sure what else we need here?

Reply:
This comment points toward basically [what I mentioned in this PR comment](https://github.com/mozilla/experimenter/pull/5852#issuecomment-868895053) - we don't set `is_paused=True` here, just a LIVE->LIVE status change that gets interpreted as a request to pause.